src/wasm-binary.h

```diff
@@ -381,7 +381,9 @@ enum EncodedType {
   v128 = -0x5, // 0x7b
   // elem_type
   AnyFunc = -0x10, // 0x70
-  // reference type
+  // opaque reference type
+  anyref = -0x11, // 0x6f
```

Comment:
I boosted this [from wabt](https://github.com/WebAssembly/wabt/blob/58e59b626b8fcf7973316eb84270c1887a8ac1f0/src/common.h#L216) because I couldn't find the values in the proposal doc.

Reply:
This value looks correct, but anyway just in case you'd like to see the current version of the reference types spec, it is here: https://webassembly.github.io/reference-types/core/_download/WebAssembly.pdf
And the repo is here: https://github.com/WebAssembly/reference-types/tree/master/document